### PR TITLE
first UI test case

### PR DESCRIPTION
**Pull request by noahsug**
Test UI rendering via karma / mocha / sinon / chai / enzyme.

 + fixed the demo example.

### CODE REVIEW

File: example/schema/example.js [hunk 1]
```diff
@@ -118,19 +118,6 @@ export const exampleDataType = parseField(DATA, {
   }
 });
 
-/*
```
Comment (noahsug) [inline example/schema/example.js]: why you have so much commented out code??

File: karma.conf.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+      module: {
+        rules: [
+          {
+            test: /\.jsx?$/,
+            loader: 'babel-loader',
+            include: [
+              path.join(__dirname, 'src'),
+              path.join(__dirname, 'test'),
+            ],
+            query: {
```
Comment (noahsug) [inline karma.conf.js]: probably should reduce this webpack duplication, but I was lazy. Could add in another change.

File: example/schema/example.js (hunk 1 above)
Comment (DeadHeadRussell) [inline example/schema/example.js]: Uh, you didn't see anything...

But these were parts of the example that I hadn't re-implemented yet and forgot to take care of.

File: karma.conf.js (hunk 2 above)
Comment (DeadHeadRussell) [inline karma.conf.js]: Yeah, a webpack / build refactor change would be nice at some point.  No need for it to be done now though.

File: karma.conf.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+    port: 9876,
+
+    colors: true,
+
+    logLevel: config.LOG_INFO,
+
+    autoWatch: false,
+
+    browsers: ['Chrome'],
```
Comment (DeadHeadRussell) [inline karma.conf.js]: You're on Mac, right?  If you haaaapen to be on Windows, could you add IE 11 (and 10 via emulation in 11) to this?  If not, I'll take care of it.

File: test/react/components/label_spec.jsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+import {shallow} from 'enzyme';
+import moment from 'moment';
+
+import 'formatron/types';
+import {Label} from 'formatron/react';
+
+chai.use(chaiEnzyme());
+
+describe('Form', () => {
+  it('renders without exploding', () => {
```
Comment (DeadHeadRussell) [inline test/react/components/label_spec.jsx]: `exploding` lol

File: karma.conf.js (hunk 3 above)
Comment (noahsug) [inline karma.conf.js]: Added! I am indeed on mac so I can't test that it works, but it "should"

File: example/schema/example.js (hunk 1 above)
Comment (noahsug) [inline example/schema/example.js]: Ooo I see, well we have a record of them in source control if we ever want to add them back.

File: package.json [hunk 5]
```diff
@@ -6,6 +6,8 @@
   "scripts": {
     "test": "karma start",
     "test:watch": "karma start --no-single-run --auto-watch",
+    "test:mac": "karma start --browsers Chrome",
+    "test:mac:watch": "karma start --browsers Chrome --no-single-run --auto-watch",
```
Comment (DeadHeadRussell) [inline package.json]: Yooo, you should totally do something like the following in the karma config so we don't have to have to ways to launch tests:

```
const isWin = /^win/.test(process.platform);
...
const browsers = isWin ? ['IE', 'Chrome'] : ['Chrome']
```

Comment (noahsug) [inline package.json]: Oh damn, that's much better.

Done!

**Pull request merged**